src/components/WebexActivity/WebexActivity.test.js

```diff
@@ -1,58 +1,49 @@
 import React from 'react';
 
-import {PeopleJSONAdapter, ActivitiesJSONAdapter} from '../../adapters';
-import {activities, people} from '../../data';
+import jsonData from '../../data';
 
 import WebexActivity, {Header, formatMessageDate} from './WebexActivity';
 
 jest.mock('../hooks/useActivity');
 jest.mock('../hooks/usePerson');
+jest.mock('../hooks/useAdapter');
 
 describe('Webex Activity component', () => {
-  let adapters;
-
-  beforeEach(() => {
-    adapters = {
-      peopleAdapter: new PeopleJSONAdapter(people),
-      activitiesAdapter: new ActivitiesJSONAdapter(activities),
-    };
-  });
-
   describe('Header component snapshot', () => {
     test('matches snapshot with "default" props', () => {
       expect(
-        shallow(<Header personID="default" adapter={adapters.peopleAdapter} created={activities.default.created} />)
+        shallow(<Header personID="default" created={jsonData.activities.default.created} adapter={jsonData.people} />)
```

Comment:
Is `jsonData.people` an adapter?

Reply:
no. it's an object to our unit test.

Reply:
I see why this is off, we shouldn't be passing an adapter to `Header` anymore, it need it for the `WebexAvatar`, but `WebexAvatar` has its own adapter now.

Reply:
yeah. I took care of it.